test/clipping.jl

```diff
@@ -5,7 +5,7 @@ sess = TensorFlow.Session(TensorFlow.Graph())
 a_raw = rand(10, 10)
```

Comment:
These tests need a bit of tidying.
Not related to this PR,
but at some point I will go through and give them `testset`s and remove excess `TensorFlow.foo` module names.